src/test/resources/TestNG.xml

```diff
@@ -7,10 +7,10 @@
         <classes>
 
 
-            <class name="Hmwk20">
-<!--            <methods>-->
-<!--                <include name=""/>-->
-<!--            </methods>-->
+            <class name="Hmwk21">
+            <methods>
+                <include name="RenamePlaylist"/>
+            </methods>
```

Comment:
Here's an example:

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >

<suite name="RenamePlaylist">
  <parameter name="BaseURL" value="https://bbb.testpro.io/" />
  <test name="renamePlaylist" preserve-order="false">
    <classes>
      <class name="NewNamePlayListTest">
      </class>
    </classes>
  </test>
</suite>
```

![image](https://user-images.githubusercontent.com/104761482/200262343-d16b3fdc-01d0-4999-87c0-24cea4c1ac00.png)

![image](https://user-images.githubusercontent.com/104761482/200262404-4b26469e-1fbe-4165-9b22-fdcdd16cfd4d.png)

Also, you can use run tests with classes or methods 

here is an example

![image](https://user-images.githubusercontent.com/104761482/200262750-3c8dbacd-0834-42aa-a3db-2e233e819a47.png)

Steps:
File > Setting (Ctrl+Alt+S)
Build, Execution, Deployment > Build Tools > gradle
Run Tests using: Intellij IDEA

![image](https://user-images.githubusercontent.com/104761482/200262588-52f0c4eb-86bf-44dc-9bac-dbab39c686dd.png)